src/components/media/media.scss

```diff
@@ -86,6 +86,13 @@
   }
 }
 
+[class*='media--'] {
```

Comment:
Could this just be `.media`?

Reply:
[It can!](https://github.com/uiowa/uids/pull/845/commits/1d5e9eb7894f4020f28938adb07147aefc370d90)